A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1597,29 +1597,15 @@ var A320_Neo_UpperECAM;
         getN1GaugeValue() {
             const engineId = (this.index + 1);
             const value = SimVar.GetSimVarValue("ENG N1 RPM:" + engineId, "percent");
-            const throttle = this.deltaThrottlePosition;
-            const currentThrottlePosition = this.getThrottlePosition(this.index);
-            this.deltaThrottlePosition = currentThrottlePosition;
-            if (value < A320_Neo_UpperECAM.Definitions.MIN_GAUGE_N1 || currentThrottlePosition > value) {
-                return value;
-            } else if (value > this.getThrottlePosition(this.index)) {
-                return (currentThrottlePosition > A320_Neo_UpperECAM.Definitions.MIN_GAUGE_N1) ? currentThrottlePosition : A320_Neo_UpperECAM.Definitions.MIN_GAUGE_N1;
-            } else if (currentThrottlePosition > throttle && value < currentThrottlePosition) {
-                return value;
-            } else if (currentThrottlePosition > A320_Neo_UpperECAM.Definitions.MIN_GAUGE_N1) {
-                return currentThrottlePosition;
-            } else {
-                return value;
-            }
-
+            return value;
         }
         getN1GaugeThrottleValue() {
             const throttle = Math.abs(Simplane.getEngineThrottleCommandedN1(this.index));
             const value = throttle * A320_Neo_UpperECAM.Definitions.THROTTLE_TO_N1_GAUGE;
             return value;
         }
         getThrottlePosition() {
-            return Simplane.getEngineThrottle(this.index);
+            return Math.abs(Simplane.getEngineThrottleCommandedN1(this.index));
```

Comment:
Why abs? can this value get negative?
May use `.toFixed(0)` to round.

Reply:
Why would you use toFixed to round? As far as I know math.round is way faster, and more expressive to what you are doing

Reply:
dunno, new to js, have seen it all over the place, but abs != round

Reply:
I suspect the throttle value goes negative on reverse thrust but would need to check. This is original Asobo code I think.